jdaviz/configs/mosviz/mosviz.yaml

```
@@ -7,7 +7,7 @@ settings:
     tab_headers: false
   context:
     notebook:
-      max_height: 725px
+      max_height: 850px
```

```suggestion
      max_height: 860px
```
Slightly squished at the bottom still at 850.